madcars/runners/localrunner.py

```diff
@@ -7,12 +7,12 @@
 import os
 import sys
 
-maps = ['PillMap']#,'PillHubbleMap', 'PillHillMap', 'PillCarcassMap', 'IslandMap', 'IslandHoleMap']
-cars = ['Buggy']#, 'Bus', 'SquareWheelsBuggy']
-games = [','.join(t) for t in product(maps, cars)]*101
-cur_dir = os.path.dirname(os.path.basename(__file__))
+maps = ['PillMap','PillHubbleMap', 'PillHillMap', 'PillCarcassMap', 'IslandMap', 'IslandHoleMap']
+cars = ['Buggy', 'Bus', 'SquareWheelsBuggy']
+games = [','.join(t) for t in product(maps, cars)]*10
```

Comment:
I suggest having 2 games only, so the agent could always play vs itself. In the current setup it plays with not updating enemy so the sequences of actions will be almost the same. It is like having bigger learning rate.

Reply:
Makes sense. But let's instead do weight sharing to fix this. Leaning towards having multiple maps. Not a blocker for me, though, can revert back

Reply:
If you you are going to have weight sharing you probably want to have 100 games, as it was earlier. I do not see the reason to have 10. It should be 2 or the big number, but not something in the middle

Reply:
This is actually a big number, 10 games multiplied by a product of maps and cars

Reply:
you could have bigger, but ok, up to you